lib/ansible/modules/monitoring/datadog_monitor.py

```diff
@@ -132,6 +132,11 @@
           - Effective for sparse values.
         version_added: "2.7"
         type: str
+    include_tags:
+        description:
+          - Whether notifications from this monitor automatically inserts its triggering tags into the title.
+        version_added: "2.9"
+        type: bool
```

Comment:
please add the default 

~~~diff
         type: bool
+        default: yes
~~~

Reply:
Ah, yes. Thank you!